hybrasyl/Objects/Creature.cs

```diff
@@ -357,6 +357,20 @@ public void SendAnimation(ServerPacket packet)
             }
         }
 
+        public void SendCastLine(ServerPacket packet)
+        {
+            Logger.InfoFormat("SendCastLine");
+            Logger.InfoFormat($"SendCastLine byte format is: {BitConverter.ToString(packet.ToArray())}");
```

Comment:
`Logger.Debug` imo, we don't have to use `Format` with C# 6, and info is not appropriate for these types of messages (I know we are oververbose elsewhere, needs to be fixed moving forward)